corehq/apps/app_manager/models.py

```diff
@@ -1743,7 +1743,8 @@ class CustomIcon(DocumentSchema):
     One can set either a simple text to display or
     an xpath expression to be evaluated for example count of cases within.
     """
-    form = StringProperty()
+
+    form = "badge"  # form is always badge
```

Comment:
Seems like a dumb question, but if a document is saved in Couch (for example) with the value of this attribute set to something other than "badge", what will the value of this `form` attribute be when JSON is loaded and passed to `CustomIcon.wrap(json_data)`?

Is this object ever even saved to Couch and/or populated with `.wrap(data)`? If not, maybe this does not matter.

Reply:
badge is the only correct value for this, if anything else was used, it would not have worked anyway. So if someone messed around, we can ignore that.